src/parser/dep.fs

```diff
@@ -330,8 +330,9 @@ let collect_one (verify_flags: list<(string * ref<bool>)>) (verify_mode: verify_
     | Pragma _ ->
         ()
     | TopLevelModule lid ->
-        raise (Err (Util.format1 "Automatic dependency analysis demands one \
-          module per file (module %s not supported)" (string_of_lid lid true)))
+        ()
+//        raise (Err (Util.format1 "Automatic dependency analysis demands one \
+//          module per file (module %s not supported)" (string_of_lid lid true)))
```

Comment:
Sure. Where's a good place to add this invariant, Parseit.parse? Or parse.fsy's inputFragment itself?